src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/helpers/DirZipper.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.helpers;
+
+import com.google.common.io.Files;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Stream;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+import static com.google.common.io.Resources.getResource;
+import static java.util.stream.Collectors.toList;
+
+public class DirZipper {
```

Comment:
I would be happy if you could please rename this as `DirectoryZipper`

Reply:
done